### PR TITLE
Updated workflow, page for each step of the workflow, new pics and videos, change in the navigation bar

### PR DESCRIPTION
**Pull request by JuliaSifOlafs**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**/**](https://docs-6bbnxw0om.avo-dev.app/) | [**/workflow/implement**](https://docs-6bbnxw0om.avo-dev.app/workflow/implement) |
|:-:|:-:|
|<a href="https://docs-6bbnxw0om.avo-dev.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/ePv2K4z0ecpik_6kGauOlPrFg7hvOFMZPNQCXD9oQxhLB8f8emY04382OCbFMm3e6EbQ5Fc698VoCzZLlPsUAA.png">(view full size)</a> |<a href="https://docs-6bbnxw0om.avo-dev.app/workflow/implement">Screenshot of /workflow/implement</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/TIFHSHiV_Vmn8gqi1inaRZbpW78h94x5NDBLWLKYbk_Rb-GePp4XfIbDixfgI6vhKGDKE2FHZjt8Sdb6FSEYTw.png">(view full size)</a> |



| [**/workflow/merge-publish**](https://docs-6bbnxw0om.avo-dev.app/workflow/merge-publish) | [**/workflow/overview**](https://docs-6bbnxw0om.avo-dev.app/workflow/overview) |
|:-:|:-:|
|<a href="https://docs-6bbnxw0om.avo-dev.app/workflow/merge-publish">Screenshot of /workflow/merge-publish</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/FtYLE1vw3_MWcQ1QaXMMybIKZKHu9GZoRAWS6J5jfZ6A_Wdck0_5txbcDKNOvz8pycS9E35AwWkc8TU-umXYGg.png">(view full size)</a> |<a href="https://docs-6bbnxw0om.avo-dev.app/workflow/overview">Screenshot of /workflow/overview</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/rib0_vcQYfa-olirwLy0Qq56ahgxnCAItnsQjPly3piIkE1rJw_rPgHx_isw3-5-vpuEcW6Vblf-3MH8CgXYNw.png">(view full size)</a> |



| [**/workflow/plan**](https://docs-6bbnxw0om.avo-dev.app/workflow/plan) | [**…request-implementation**](https://docs-6bbnxw0om.avo-dev.app/workflow/request-implementation) |
|:-:|:-:|
|<a href="https://docs-6bbnxw0om.avo-dev.app/workflow/plan">Screenshot of /workflow/plan</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/3rE3zF2R_Vnv31l5ZoFoF1AIRHv4JaGB6Ft7q4oU7wHdD4hlZwxLNN7_LMU5WrKp5WAoTRY_UI3JdHbqpelXtA.png">(view full size)</a> |<a href="https://docs-6bbnxw0om.avo-dev.app/workflow/request-implementation">Screenshot of /workflow/request-implementation</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/HBW5VypQ_ozw-inpDs8XRMTLWYB8wzATO35hW0cSFI_1DTn4UH85j4t9FooVQP1buRTqEghKydftrngl47BS3Q.png">(view full size)</a> |



And 2 other routes:
- [**/workflow/review**](https://docs-6bbnxw0om.avo-dev.app/workflow/review)
- [**/workflow/validate**](https://docs-6bbnxw0om.avo-dev.app/workflow/validate)



Commit 6c1be8abe8e2b0fd85e9f03e5dd9890247c655cd (https://docs-6bbnxw0om.avo-dev.app).

**Comment by logason** [comment]
@JuliaSifOlafs could you update the PR title and description to better represent the changes happening in this PR? Looks like we've expanded it from just the "Overview page of the workflow" to docs for all the steps?

File: pages/workflow/implement.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
```
Comment (logason) [inline pages/workflow/implement.mdx:1]: Do we have any opportunities to add screenshots or videos here, to make this page pop a bit?

Inspo, the Avo Functions GIF we have in this deck: https://docs.google.com/presentation/d/1KsyBWXSS2g3AJrETL4-D0ZEdofZtn7IUipPXx3j7VCw/edit#slide=id.g1369a7f867c_0_1036

File: pages/workflow/overview.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+5. **Validate** implementation using Avo
+6. **Merge** Avo branch and **Publish** schema to downstream schema registries
+
+<center>
+  <img
+        width="100%"
+        src={require('../../images/avo-workflow.png')}
+        alt="The Avo Workflow: Plan, review, implement, validate and publish your tracking changes"
+  />
+</center>
```
Comment (logason) [inline pages/workflow/overview.mdx:26]: HMW use the same Zoom image thing as we have on the docs front page today 🤔

File: pages/workflow/plan.mdx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+## 1.2 Create a branch in Avo
+
+_Role: Product or Data. (This is also sometimes done by engineering, particularly when the engineering team is product driven.)_
+
+The first thing you do in Avo, for a new feature or iteration, is to create a new branch.
+
+A branch allows you to draft changes to the tracking plan in isolation, without impacting the main branch, which contains the single source of truth of what’s currently being tracked.
+
+To create a new branch, first make sure you are on the main branch. Head to the <Link passHref href="/workspace/branches#branches-overview"><a>Branches overview</a></Link> screen by clicking the Avo logo, and from there click “Create branch”.
```
Comment (logason) [inline pages/workflow/plan.mdx:32]: This doc does not match the video below. We should make sure they are consistent. Should we update the text here to match what is happening in the video @JuliaSifOlafs ?

File: pages/workflow/validate.mdx [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+# Validate implementation with Avo Inspector
+
+For every source that has the Inspector installed, you can validate if the implementation is according to the tracking plan using the Inspector dashboard and/or the Inspector implementation in your tracking plan.
+
+If you don’t have the Inspector installed yet, you can learn more about how to get started with the Inspector here:
+
+> [Use Inspector to improve your tracking plan health](https://www.avo.app/docs/data-design/start-using-inspector)
+
+
+**Validate with the Inspector dashboard**
```
Comment (logason) [inline pages/workflow/validate.mdx:22]: We should use ## or ### for all subtitles here so they show up in the page overview

Screenshot 2022-07-18 at 17 58 47

```suggestion
### Validate with the Inspector dashboard
```

File: pages/workflow/implement.mdx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+You can continue implementing your tracking code like you are used to using analytics SDKs from analytics platforms (like Mixpanel, PostHog or Amplitude), SDKs from CDPs (like Rudderstack, Segment or mParticle), or using internal built SDKs or APIs.
+
+A typical implementation workflow looks like this:
+1. Open a git branch to implement your analytics tracking changes on
+2. Follow the implementation instructions provided on the branch review screen in Avo for what needs to be done
+
+When using the implementation instructions from Avo, we recommend checking out our guide on how to read the implementation instructions:
+
+> [How to read the implementation diff](https://www.avo.app/docs/implementation/read-implementation-diff)
```
[the remark this answers is not in the document]
Comment (JuliaSifOlafs) [reply pages/workflow/implement.mdx:23]: nice changed this

File: .spelling [hunk 6]
```diff
@@ -419,3 +419,18 @@ SHA-256
...
+2.review
+Asana
+ - pages/workflow/implement.mdx
+Rudderstack
+ - pages/workflow/plan.mdx
+1.plan
+event.png
+ - pages/workflow/request-implementation.mdx
+3.request
+ - pages/workflow/validate.mdx
```
[the remark this answers is not in the document]
Comment (JuliaSifOlafs) [reply .spelling:433]: When I did the spellcheck those came as an error so I added them to be ignored using the fix spellcheck command.

File: pages/workflow/overview.mdx (hunk 2 above)
Comment (JuliaSifOlafs) [reply pages/workflow/overview.mdx:26]: zoom is now on every image.

File: pages/workflow/overview.mdx [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+<Zoom>
+  <center>
+    <img
+          width="100%"
+          src={require('../../images/workflow/overview/avo-workflow.png')}
+          alt="The Avo Workflow: Plan, review, implement, validate and publish your tracking changes"
+    />
+  </center>
+</Zoom>
```
Comment (logason) [inline pages/workflow/overview.mdx:31]: It looks like there is something funky going on here, I can't click the image, but there is this tiny empty button right below it which expands it, but it expands in a weird place.

Screenshot 2022-07-21 at 22 12 02

If we can't easily fix I suggest we ship without the zoom

File: pages/workflow/overview.mdx [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+The Avo workflow is the recommended approach for a team working on an analytics release. The workflow is helpful both when new analytics tracking is being implemented, as well as when existing tracking is being updated.
+
+The Avo workflow for analytics releases is inspired by the general best practices of software development using branched change management, peer-reviews, and approval workflows.
+
+1. **Plan** analytics update
+2. **Review** suggestions
+3. **Request** implementation instructions with developers
+4. **Implement** analytics events
+5. **Validate** implementation
+6. **Merge** Avo branch and **Publish** schema to downstream schema registries
```
Comment (logason) [inline pages/workflow/overview.mdx:21]: Can we link to each step here?

File: pages/workflow/overview.mdx [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+## Best practices for a streamlined analytics release workflow
+
+Your main branch in Avo should represent what is currently being tracked in production. As soon as event implementation is done, you merge your Avo branch into your Avo main branch.
+
+For your analytics release cycle (and Avo branches), we recommend following the same best practices as generally apply to the software development lifecycle (and git branches): 
+- Release Early, Release Often. This will streamline collaboration across teams, platforms and products of your new single-source-of-truth tracking plan. 
+- In practice, this means: Keep Avo branches small and release often, rather than designing a large set of changes that should be implemented over multiple product releases. 
+
+## Ready to get going?
+>#### <Link passHref href="/workflow/plan"><a>1. Plan analytics update in Avo branch</a></Link>
```
Comment (logason) [inline pages/workflow/overview.mdx:49]: Use the new prettified link here 👯

File: pages/workflow/plan.mdx [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+1. Goals – what does success look like?
+2. Metrics – how will we measure how successful we are?
+3. Data – what analytics events do we need for our success metrics?
+
+The result of the purpose meeting is usually a draft of events that someone takes forward to the next step.
+
+ <div className={styles.row}>
+          <PageLink
+            image={AnalyticsManagerIcon}
+            title="Purpose Meeting"
```
Comment (logason) [inline pages/workflow/plan.mdx:32]: I think it's good to set the stage that this takes you out of the docs and into the blog
```suggestion
            title="blog Post: Purpose Meeting"
```

File: pages/workflow/plan.mdx [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+
+<Zoom>
+  <center>
+    <img
+          width="70%"
+          src={require('../../images/workflow/1.plan/create-new-branch.png')}
+          alt="create a new branch from the navigation bar"
+    />
+  </center>
+</Zoom>
```
Comment (logason) [inline pages/workflow/plan.mdx:58]: Same zoom weirdness here as on the overview page 🤔

File: pages/workflow/plan.mdx [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+          alt="even definition"
+    />
+  </center>
+</Zoom>
+
+_Example: Event definition_
+
+
+## What’s next?
+>#### <Link passHref href="/workflow/review"><a>2. Review suggestions in Avo</a></Link>
```
Comment (logason) [inline pages/workflow/plan.mdx:104]: We should upgrade this one to prettier link ✨

File: pages/workflow/review.mdx [hunk 13]
```diff
@@ -0,0 +1,147 @@
...
+
+<video
+  controls
+  loop
+  muted
+  width="100%"
+  autoPlay
+  src={require('../../images/workflow/2.review/add-collaborators.mp4')}
+  alt="Add Collaborators"
+/>
```
Comment (logason) [inline pages/workflow/review.mdx:59]: Can we make this video have the same indent as the bullet it's attached to? Maybe just removing the newline between them fixes it?
Screenshot 2022-07-21 at 22 18 02

File: pages/workflow/validate.mdx [hunk 14]
```diff
@@ -0,0 +1,257 @@
...
+
+<Zoom>
+  <center>
+    <img
+          width="100%"
+          src={require('../../images/workflow/5.validate/dev-stag-production.png')}
+          alt="validate on inspector dashboard"
+    />
+  </center>
+</Zoom>
```
Comment (logason) [inline pages/workflow/validate.mdx:57]: Can we make the screenshot follow the indent of the bullet it's attached to here? Same for the screenshot right below this one

File: pages/workflow/validate.mdx [hunk 15]
```diff
@@ -0,0 +1,257 @@
...
+            callToAction={
+              new CallToAction('https://www.avo.app/docs/workspace/tracking-plan/implementation-status#avo-functions-implementation-status')
+            }
+          />
+  </div>
+
+
+## What’s next?
+
+>#### <Link passHref href="/workflow/merge-publish"><a>6. Merge branch and Publish</a></Link>
```
Comment (logason) [inline pages/workflow/validate.mdx:256]: Upgrade this link to the new version! (PageLink)

File: pages/workflow/merge-publish.mdx [hunk 16]
```diff
@@ -0,0 +1,60 @@
...
+          />
+</div>
+
+## You just completed the final step of the Avo workflow!
+
+Congrats, by completing this step you’ve completed the Avo workflow for an analytics release! 🎉
+
+Head back to the Avo Workflow overview page:
+
+>#### <Link passHref href="/workflow/overview"><a>The Avo workflow</a></Link>
```
Comment (logason) [inline pages/workflow/merge-publish.mdx:58]: Replace with PageLink ✨

File: pages/workflow/review.mdx [hunk 17]
```diff
@@ -0,0 +1,147 @@
...
+
+To see more details about the items being changed you can click into each item on the review screen. You can add comments directly to each item. All comments made on the branch will show up on the review screen.
+
+<video
+  controls
+  loop
+  muted
+  width="100%"
+  autoPlay
+  src={require('../../images/workflow/2.review/commenting-in-an-item.mp4')}
```
Comment (logason) [inline pages/workflow/review.mdx:121]: This mp4 is 1mb, can we compress the mp4s somehow before to reduce the file sizes people have to download while reading the docs?

**Review by logason — COMMENTED** [review]
Looks great to me @JuliaSifOlafs, awesome work here!

Added few comments I think we should address before shipping. The biggest one is around the <Zoom /> component in the md files, it looks like it isn't working as expected in there, so I'd like to suggest we drop it from those for now.

File: pages/workflow/overview.mdx (hunk 7 above)
Comment (JuliaSifOlafs) [reply pages/workflow/overview.mdx:31]: Okay, this is super weird, when I run this on my localhost the zoom works perfectly and this tiny box at the bottom is not there. I don't know why this appears like this now.

File: pages/workflow/merge-publish.mdx [hunk 18]
```diff
@@ -0,0 +1,70 @@
...
+              new CallToAction('https://www.avo.app/docs/workspace/tracking-plan/publishing')
+            }
+          />
+</div>
+
+## You just completed the final step of the Avo workflow!
+
+Congrats, by completing this step you’ve completed the Avo workflow for an analytics release! 🎉
+
+Head back to the Avo Workflow overview page:
```
Comment (alex-tpom6oh) [inline pages/workflow/merge-publish.mdx:56]: This line and the link below seems a bit redundant?

**Review by alex-tpom6oh — APPROVED** [review]
Great content 😄

**Pull request merged**